core-bundle/src/EventListener/AddCustomRegexp/CustomRgxpListener.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\EventListener\AddCustomRegexp;
```

Comment:
Maybe use a different name?

```suggestion
namespace Contao\CoreBundle\EventListener\WidgetRegexp;
```